media/js/src/AssetFilter.jsx

```diff
@@ -5,54 +5,64 @@ import Select from 'react-select';
 export default class AssetFilter extends React.Component {
     constructor(props) {
         super(props);
-        this.state = {
-            selectedOwner: 'all'
-        };
 
         this.handleOwnerChange = this.handleOwnerChange.bind(this);
         this.handleTitleSearch = this.handleTitleSearch.bind(this);
+
+        this.allOption = {
+            value: 'all',
+            label: 'All Class Members'
+        };
     }
     handleTitleSearch(e) {
         if (e.key === 'Enter') {
             this.props.handleTitleFilterSearch();
         }
     }
     handleOwnerChange(e) {
-        this.setState({selectedOwner: e.value});
```

Comment:
Sorry to be a PITA, but did you mean to pull this?

Reply:
yes - i'm no longer using this state value.